integrations/convex/manifest.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "Convex",
+  "version": "0.0.1",
+  "overview": "Sync your LaunchDarkly feature flags to your Convex deployment.",
+  "description": "Convex is the open-source backend for application builders. An all-in-one platform with thoughtful, product-centric APIs.",
+  "author": "Convex",
+  "supportEmail": "support@convex.dev",
+  "links": {
+    "site": "https://convex.dev",
+    "privacyPolicy": "https://convex.dev/privacy"
+  },
+  "categories": ["infrastructure"],
+  "icons": {
+      "square": "assets/images/square.svg",
+      "horizontal": "assets/images/horizontal.svg"
+    },
```

Comment:
I think prettier is getting tripped up on the spacing here. Can you do me a favor and run `npm run prettier:write`